.github/workflows/linter.yml

```diff
@@ -208,6 +214,9 @@ jobs:
           - examples/piggy-bank/part2/Cargo.toml
           - examples/auction/Cargo.toml
           - examples/erc721/Cargo.toml
+          - examples/cts1-wgtu/Cargo.toml
+          - examples/cts1-nft/Cargo.toml
+          - examples/cts1-multi/Cargo.toml
```

Comment:
Also, please add these examples to the examples/README.md file.

Reply:
Already there